iamproperty/iamproperty.github.io
# Ds 76/collapsible content

## Summary of Changes:
1. Minor CSS updates
2. Buttons doc page updated

## Rationale
Display a collapsible bit of content

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /elements/buttons (bottom of page)

## Ticket(s)
DS-76

## Notes
